crates/primitives/src/result.rs

```diff
@@ -24,7 +24,11 @@ pub enum ExecutionResult {
         output: Output,
     },
     /// Reverted by `REVERT` opcode that doesn't spend all gas.
-    Revert { gas_used: u64 },
+    Revert {
+        gas_used: u64,
+        logs: Vec<Log>,
```

Comment:
Logs are getting reverted, they will always be empty, it does not make sense to have them here: https://github.com/bluealloy/revm/blob/ba393d75cf12ec0b1446fd5ce147b338de6bb662/crates/revm/src/evm_impl.rs#L738-L742

Reply:
Removed in latest commit